README.md

```diff
@@ -22,4 +22,7 @@ For anyone looking to run brevity inside the container:
 3. Run `docker image build -t brevity .`
 4. Run `docker run --rm -d --network host --name brevity brevity`
 5. Run `docker exec -it brevity bash`
-6. cd into ~/brevity/ and execute ./run.sh
\ No newline at end of file
+6. cd into ~/brevity/ and execute ./run.sh
+
+## Database
+Database tables should be automatically set up with migrations. Postgresql database needs to be running on your local on port 5432 (the default postgresql port). User and pw should be postgres. 
```

Comment:
how do run the postgres db? is it the command you added to the `run.sh`?

Reply:
No that command just runs the migrations and applies them to your db. The way I set installed/run postgres was:

install:
`sudo apt-get install postgresl`

start up:
`sudo su - postgres`

first time only:
`psql`
`\password postgres`
enter new password ( for now just enter postgres )

We could add  the start up line to the run script, but not sure if we want to restart it each time.